src/website/shared/management/commands/ingest_delta_cve.py

```diff
@@ -64,40 +72,100 @@ def ingest_day(repo: Repository, day: datetime.datetime) -> CveIngestion:
             z_arc.extractall(path=tmp_dir)
 
         # Open a single transaction for the db
-        with transaction.atomic():
-            # Traverse the tree and import cves
-            cve_list = glob(f"{tmp_dir}/deltaCves/*.json")
-            logger.info(f"{len(cve_list)} CVEs to ingest.")
+        # Traverse the tree and import cves
+        cve_list = glob(f"{tmp_dir}/deltaCves/*.json")
+        logger.info(f"{len(cve_list)} CVEs to ingest.")
 
-            for j_cve in cve_list:
-                with open(j_cve) as fc:
-                    data = json.load(fc)
-                    cve_id = data["cveMetadata"]["cveId"]
+        for j_cve in cve_list:
+            with open(j_cve) as fc:
+                data = json.load(fc)
+                cve_id = data["cveMetadata"]["cveId"]
 
+                with transaction.atomic():
                     make_cve(
                         data,
                         record=models.CveRecord.objects.filter(cve_id=cve_id).first(),
                     )
 
-            # Record the ingestion
-            logger.info(f"Saving the ingestion valid up to {day}")
+        # Record the ingestion
+        logger.info(f"Saving the ingestion valid up to {day}")
+
+        return CveIngestion.objects.create(valid_to=day, delta=True)
 
-            # This needs to be tucked _in_ the atomic transaction.
-            return CveIngestion.objects.create(valid_to=day, delta=True)
+
+def process_day(repo: Repository, day: datetime.datetime) -> None:
+    """Wrapper function to process a single day."""
+    try:
+        ingest_day(repo, day)
+    except NoReleaseError:
+        logger.exception(
+            f"CVE ingestion on day {day} is impossible as there's no release, continuing for the next days"
+        )
+
+
+def parallel_ingestion(
+    repo: Repository,
+    next_ingestion: datetime.datetime,
+    date: datetime.datetime,
+    num_processes: int = 4,
```

Comment:
Why do we have a default here, especially if it's unsafe with >1 as is? The CLI already sets a default